privacyidea/lib/caconnectors/localca.py

```diff
@@ -402,15 +401,9 @@ def sign_request(self, csr, options=None):
                 csr_obj.get_subject(), file_extension="pem")
             #csr_extensions = csr_obj.get_extensions()
         csr_filename = csr_filename.replace(" ", "_")
-        if type(csr_filename) == str:
-            csr_filename = csr_filename.decode("utf-8")
-        csr_filename = csr_filename.encode("ascii", "ignore")
```

Comment:
So why would this suddently be not necessary under python 2.7 anymore?

Reply:
You are right, i misinterpreted this. Since we are using the updated path later on there should be no trouble. I'll fix this.

Reply:
fixed in 1fb2839

Reply:
I think we also should use ``to_unicode`` instead of decode?